registry/coder-labs/modules/codex/scripts/install.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
```

Comment:
why not load the bash env here?

Reply:
Depending on the template `npm` and other tools may become visible only after the bash env is sourced. Feels safer to source it.